modular_lumos/code/modules/mob/lumos_emotes.dm

```diff
@@ -85,7 +85,129 @@
 
     user.nextsoundemote = world.time + 20
     playsound(user, pickweight(list(
-		'modular_lumos/sound/effects/fart.ogg' = 50, 
+		'modular_lumos/sound/effects/fart.ogg' = 50,
 		'modular_lumos/sound/effects/fart2.ogg' = 30,
 		'modular_lumos/sound/effects/fart3.ogg' = 5,
 		'modular_lumos/sound/effects/fart4.ogg' = 10)), 50, FALSE)
+    var/brap = rand(1,5)
+    user.atmos_spawn_air("miasma=[brap];TEMP=310.15")
```

Comment:
this was removed, why bring itback?

Reply:
yea, this needs to stay out. I removed it because people are gamers.